editor/templates/editoritem/preview.html

```diff
@@ -54,19 +58,48 @@ <h1 class="description">
 
 
         <script>
-var share_button = document.getElementById('share');
-var embed_display = document.getElementById('embed-url');
-share_button.addEventListener('click',function() {
-    embed_display.classList.toggle('shown');
-    embed_display.focus();
-});
+
+            var share_button = document.getElementById('share');
+            var embed_display = document.getElementById('embed-url');
+            var copy_text = document.getElementById('copy-text');
+            var close_button = document.getElementById('close-button');
+
+            /* To toggle the embedded URL area */
+            share_button.addEventListener('click',function() {
+                embed_display.classList.toggle('shown'); 
+                embed_display.focus();
```

Comment:
The `#embed-url` tag doesn't appear in the tab order, so it can't be focused. You wouldn't want to focus it anyway - it's not interactive.